src/test/kotlin/lotto/domain/LottoTest.kt

```diff
@@ -0,0 +1,31 @@
+package lotto.domain
+
+import lotto.domain.value.HitLotto
+import lotto.domain.value.LottoNumber
+import lotto.strategy.TestStrategy
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+internal class LottoTest {
+    private val lotto = Lotto(TestStrategy())
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
    private val lotto = Lotto(1, 2, 3, 4, 5, 6)
```

Reply:
처음에는 생성자의 수를 늘리면 validation 로직이 추가되어서 사용하기 더 어려워지는 것은 아닌지 걱정했는데, 여러 생성자를 만들어서 제공하면 사용하는 쪽에서 다양한 방법으로 객체를 사용할 수 있어서 편리하다는 생각을 했습니다.